### ISSUE TITLE
500 ActionView::MissingTemplate

### ISSUE THREAD

**Issue by Flaburgan** [diaspora/diaspora#6833]
I still have some of these errors every now and then, on both diaspora-fr (running last develop) and framasphere (running last master). Does anybody know where that comes from? Sometimes it's from /posts, sometimes from /tags. Looks like it always has the _ parameter.

```
[2016-05-13T16:17:46] INFO  PID-823 TID-1889472280980 Rails: Started GET "/posts/393563/interactions?_=1444953600000" for X.X.X.X at 2016-05-13 16:17:46 +0200
[2016-05-13T16:17:46] INFO  PID-823 TID-1889472280980 ActionController::Base: Processing by PostsController#interactions as */*
[2016-05-13T16:17:46] INFO  PID-823 TID-1889472280980 ActionController::Base:   Parameters: {"_"=>"1444953600000", "id"=>"393563"}
[2016-05-13T16:17:46] INFO  PID-823 TID-1889472280980 ActionController::Base: Completed 500 Internal Server Error in 4ms (ActiveRecord: 0.4ms)
[2016-05-13T16:17:46] FATAL PID-823 TID-1889472280980 Rails: 
ActionView::MissingTemplate (Missing template posts/interactions, application/interactions with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:erb, :builder, :raw, :ruby, :markerb, :haml]}. 
Searched in:
  * "/home/diaspora/diaspora/app/views"
  * "/home/diaspora/diaspora/vendor/bundle/ruby/2.2.0/gems/rails_admin-0.8.1/app/views"
  * "/home/diaspora/diaspora/vendor/bundle/ruby/2.2.0/gems/kaminari-0.16.3/app/views"
  * "/home/diaspora/diaspora/vendor/bundle/ruby/2.2.0/gems/entypo-rails-3.0.0.pre.rc2/app/views"
  * "/home/diaspora/diaspora/vendor/bundle/ruby/2.2.0/gems/devise-3.5.6/app/views"
):
  app/controllers/posts_controller.rb:44:in `interactions'
  lib/rack/internet_explorer_version.rb:34:in `call'

```

**Comment by Flaburgan** [comment]
@jhass can you please tell me how I can provide more info?

**Comment by jhass** [comment]
These kind of errors are unfixable without a way to reproduce.

**Comment by denschub** [comment]
GET /posts/[ID]/interactions

without a `Content-Type`/`Accept` header. However, this is not a frontend issue, all calls I get are made by the lovely Yandex bot.

**Comment by SuperTux88** [comment]
Closed via #7085
